tree/ntupleutil/v7/test/CustomStructUtil.hxx

```diff
@@ -0,0 +1,16 @@
+#ifndef ROOT7_RNTupleUtil_Test_CustomStructUtil
+#define ROOT7_RNTupleUtil_Test_CustomStructUtil
+
+#include <string>
+#include <vector>
+
+/**
+ * Used to test importing of classes with dictionary
+ */
+struct CustomStructUtil {
+   float a = 0.0;
+   std::vector<float> v1;
```

Comment:
We might also consider adding another field of type `std::vector<std::vector<float>>`; would that work in the current implementation?

Reply:
Added nested vector to the unit test.